projects/control-service/projects/job-builder/build_image.sh

```diff
@@ -15,6 +15,10 @@ registry_type=$8
 registry_username=$9
 registry_password=${10}
 
+# Within this property docker config should be included to connect to the registry used to pull the image from.
+# it should be prefixed with a comma
```

Comment:
that is unusual, wouldn't itbe error-prone...

Reply:
Yes this is defo error prone. 
But it is providing way more flexibility to customers that I think it is worth it.